templates/typescript-angular-class.mustache

```diff
@@ -19,7 +19,7 @@ module {{moduleName}} {
 *    provide using <code>.constant('domain', '//example.com')</code> or <code>.factory('domain', function(){return '//example.com'})</code>
 */
 export class {{&className}} {
-    static $inject = ['$http', '$q', 'domain'];
+    static $inject = ['$http', '$q', '{{domainName}}'];
 
     constructor(private $http: ng.IHttpService, private $q: ng.IQService, private domain: string) {}
```

Comment:
@adrianwong81 Have you tested this?  You'll need to replace all references to `domain` with `{{domainName}}`.

...actually, I think that `domainName` is a bit misleading - it's actually the name of the variable that will be used, not the value of the domain name...